monai/apps/generation/maisi/networks/diffusion_model_unet_maisi.py

```diff
@@ -102,6 +98,8 @@ def __init__(
         cross_attention_dim: int | None = None,
         num_class_embeds: int | None = None,
         upcast_attention: bool = False,
+        include_fc: bool = True,
```

Comment:
Perhaps we should set the default value for this one to False here? Since the weight has trained without final fc.
What do you think? @guopengf @dongyang0122

Reply:
Sure, change it to False